### ISSUE TITLE
Accordion - Override button behavior inside accordion/tabs

### ISSUE THREAD

**Issue by quadrinaquinn** [USAJOBS/design-system#73]
Buttons inside the accordion/tab click events are trapped. Updated the usa-design-system-component.js code by adding new data attribute to allow the default button behavior. Updated code is currently running in test.usajobs.gov.

```
function Accordion($el) {
  var self = this;
  this.$root = $el;
  this.$root.on('click', 'button', function (ev) {
      var action = $(this).data('button-action');
      if (action != 'no-toggle') {
          var expanded = JSON.parse($(this).attr('aria-expanded'));
          ev.preventDefault();
          self.hideAll();
          if (!expanded) {
              self.show($(this));
          }
          self.jumpTo($(this));
      }     
  });
}
```

**Comment by humancompanion** [comment]
@quadrinaquinn - This is definitely a bug in the uswds. But, I think a better fix would be to either:

1. Change this function to bind to not just any old button but, a button that has an appropriate data attribute (like data-accordion-trigger or something). I suspect we have more buttons within our accordions that do something else than we have buttons that actually trigger the accordion.
2. Excise this function entirely and rely on our own. I'd like to go this route because I'd like one generic drawer/accordion JS file that handled either case (accordions only allow one section of content to be visible, drawers have no such restriction). 

Unfortunately, to get to option 2 we need to deal with the fact that uswds moved to writing all of their JS as modules that are bundled up by browserify. Then we can pick and choose which modules we want to include and thus choose to ignore their Accordion JS and write our own.

For now I will add your work around to accordion.js. But that lives in the design system, so I'll open a new bug over there.

**Comment by humancompanion** [comment]
Fixed in 8eec20e539447cb1cd663c091ff91f7ea38775fd.
